packages/fints/src/__tests__/test-pin-tan-client-e2e.ts

```diff
@@ -0,0 +1,69 @@
+import * as fs from "fs";
+import { TanRequiredError } from "../errors/tan-required-error";
+import { PinTanClient } from "../pin-tan-client";
+import { SEPAAccount } from "../types";
+
+const url = process.env.FINTS_URL;
+const name = process.env.FINTS_USER;
+const pin = process.env.FINTS_PASSWORD;
+const blz = process.env.FINTS_BLZ;
+
+const productId = "9FA6681DEC0CF3046BFC2F8A6";
+
+test.skip("get accounts", async () => {
```

Comment:
I suggest to remove the `.skip` and add another test command for jest that only tests the e2e files. Also I would appreciate some documentation on how exactly to use these tests and what for. Or simply delete this file.